sdk2ecr

```diff
@@ -0,0 +1,114 @@
+#!/bin/bash
+
+set -e -o pipefail
+shopt -qs failglob
+
+for opt in "$@"; do
+   optarg="$(expr "${opt}" : '[^=]*=\(.*\)')"
+   case "${opt}" in
+      --version=*) VERSION="${optarg}" ;;
+      --registry=*) REGISTRY="${optarg}" ;;
+      --sdk-name=*) SDK_NAME="${optarg}" ;;
+   esac
+done
+
+if [ -z "${VERSION}" ]; then
+  VERSION=$(make print-version | tail -1)
+fi
+
+for required_arg in VERSION REGISTRY SDK_NAME; do
+  [ -z "${!required_arg}" ] && echo "${required_arg} not set!" >&2 && exit 1
+done
```

Comment:
It'd also be good to add defaults for these to the `Makefile` even though we also check here. Maybe I'm weird but I would tend to read the `Makefile` ahead of time to figure out what I should specify in my `make` invocation, and wouldn't think to look into the various scripts until I hit an error.

We could default to `bottlerocket` as the SDK name and `localhost:5000` for the registry.